src/universal/components/Auth0ShowLock/Auth0ShowLock.js

```diff
@@ -27,9 +27,9 @@ export function showLock(dispatch) {
   // eslint-disable-next-line global-require
   const Auth0Lock = require('auth0-lock');
   let clientOptions = defaultClientOptions;
-  if (__PRODUCTION__) {
+  if (typeof __PRODUCTION__ !== 'undefined' && __PRODUCTION__) {
```

Comment:
`__PRDUCTION__` is guaranteed if this is built with webpack, why is this required?